website/docs/d/project.html.markdown

```diff
@@ -12,8 +12,9 @@ description: |-
 
 -> **NOTE:** Groups and projects are synonymous terms. You may find group_id in the official documentation.
 
-## Example Usage
+## Examples Usage
```

Comment:
FYI Example Usage works for more than one.   Examples Usage is a bit of an odd phrasing.

Reply:
Thank you so much :)